components/ConversationsList.tsx

```diff
@@ -30,6 +32,9 @@ const ConversationTile = ({
   const { messages } = useConversation(conversation.peerAddress)
   const latestMessage = getLatestMessage(messages)
   const path = `/dm/${conversation.peerAddress}`
+  if (!latestMessage) {
+    return <div />
```

Comment:
You should be able to return `null` here.

Also, are we sure this is going to behave the way we want for brand new conversations?

Reply:
Oh, yes, just have to set the return type to `JSX.Element | null`.

It doesn't change the behavior for brand new conversations, since conversation tiles already aren't displayed until after a message is sent. It may need to be changed if, for instance, we wanted to add a tile for a "draft" conversation that the user can click away from and come back to.